### DISCUSSION TITLE
Reopen issue that was converted to discussion

### DISCUSSION THREAD

**Discussion by marlonrichert** [community/community#85014] [Discussions]
### Select Topic Area

Product Feedback

### Body

I converted an issue to a discussion, but then later noticed that it should've been an issue anyway. Normally, closed issues can be reopened. However, when an issue is closed because it was "converted" to a discussion, then there is no way to reopen it. Please allow such issues to be reopened, just like any other closed issue.

**Comment by DavidGBrett** [comment]
This is still an problem
Discussion #2861 raised a similar problem but I don't find the solution sufficient enough.
Opening a new issue is not the same as reopening the original one - new author and no context left

Ideally the old issue should be reopened and the context from the discussion appended to it - though just showing a link to the old discussion thread could be an acceptable substitute if that's too much.
